src/Kernel.php

```diff
@@ -40,6 +44,20 @@ public function registerBundles() : iterable
         }
     }
 
+    public function process(ContainerBuilder $container) : void
+    {
+        $diffCommand = $container->findDefinition('doctrine_migrations.diff_command');
+        $doctrineItems = $container->findDefinition('libero.content_store.items');
+
+        $schema = new Definition(Schema::class);
+        $schema->setFactory([$doctrineItems, 'getSchema']);
+
+        $schemaProvider = new Definition(StubSchemaProvider::class);
+        $schemaProvider->setArgument(0, $schema);
+
+        $diffCommand->setArgument(0, $schemaProvider);
```

Comment:
Appears to not be an existing way to configure the diff command when not using the ORM. Some of these can be moved into the ContentApiBundle; will open a PR on DoctrineMigrationsBundle to add a setting.

Reply:
Opened https://github.com/doctrine/DoctrineMigrationsBundle/pull/265, used in 
a50156a.